dap_aria_mapping/getters/app_tables/change_makers.py

```diff
@@ -0,0 +1,42 @@
+from nesta_ds_utils.loading_saving.S3 import download_obj
+from dap_aria_mapping import BUCKET_NAME
+import networkx as nx
+import polars as pl
+
+def get_collaboration_network(dataset: str) -> nx.Graph:
+    """get the collaboration network for either patents or publications
+
+    Args:
+        dataset (str): "Academia" for publication network or "Industry" for patent network 
+
+    Returns:
+        nx.Graph: collaboration network
+    """
+    if dataset == "Industry":
+        return download_obj(BUCKET_NAME, "outputs/app_data/change_makers/networks/patents_institutions.pkl")
+    elif dataset == "Academia":
+        return download_obj(BUCKET_NAME, "outputs/app_data/change_makers/networks/publications_institutions.pkl")
+    else:
+        return "Not implemented"
```

Comment:
Are we sure we want to return this string, rather than adding an `assert dataset in ["Industry", "Academia"], ' The requested dataset is not implemented'` line?

Reply:
will change in 73